### PR TITLE
convert EndpointNamespace showing string rather than pointer

### LINKED ISSUE #81466: kubectl gluster EndpointNamespace not showing properly
**What happened**:

```
Source:
    Type:                Glusterfs (a Glusterfs mount on the host that shares a pod's lifetime)
    EndpointsName:       server
    EndpointsNamespace:  0xc0006a17a0
    Path:                gv0
    ReadOnly:            false
Events:                  <none>
```

**What you expected to happen**:

EndpointNamespace should be the string name of the namespace used. 

**How to reproduce it (as minimally and precisely as possible)**:

Create a glusterfs PV using an EndpointNamespace entry and run 'kubectl describe pv' on the resulting PV. 

**Anything else we need to know?**:

**Environment**:
- Kubernetes version (use `kubectl version`):
Client Version: version.Info{Major:"1", Minor:"15", GitVersion:"v1.15.2", GitCommit:"f6278300bebbb750328ac16ee6dd3aa7d3549568", GitTreeState:"clean", BuildDate:"2019-08-05T17:09:13Z", GoVersion:"go1.12.7", Compiler:"gc", Platform:"linux/amd64"}
Server Version: version.Info{Major:"1", Minor:"15", GitVersion:"v1.15.2", GitCommit:"f6278300bebbb750328ac16ee6dd3aa7d3549568", GitTreeState:"clean", BuildDate:"2019-08-05T09:15:22Z", GoVersion:"go1.12.5", Compiler:"gc", Platform:"linux/amd64"}
- Cloud provider or hardware configuration:
Brightbox
- OS (e.g: `cat /etc/os-release`):
Ubuntu 18.04.3 LTS
- Kernel (e.g. `uname -a`):
Linux bionic-vm 4.15.0-58-generic #64-Ubuntu SMP Tue Aug 6 11:12:41 UTC 2019 x86_64 x86_64 x86_64 GNU/Linux
- Install tools:
kubeadm 1.15

### PR DESCRIPTION
**Pull request by zouyee**
**What type of PR is this?**

/kind bug
/kind flake

**What this PR does / why we need it**:
convert EndpointNamespace showing string rather than pointer

**Which issue(s) this PR fixes**:

Fixes #81466

**Special notes for your reviewer**:

**Does this PR introduce a user-facing change?**:

```release-note
NONE
```

**Additional documentation e.g., KEPs (Kubernetes Enhancement Proposals), usage docs, etc.**:

```docs

```

### CODE REVIEW

**Comment by zouyee** [comment]
/sig cli
@kubernetes/sig-cli-misc

File: staging/src/k8s.io/kubectl/pkg/describe/versioned/describe.go [hunk 1]
```diff
@@ -997,12 +997,16 @@ func printGlusterfsVolumeSource(glusterfs *corev1.GlusterfsVolumeSource, w Prefi
 }
 
 func printGlusterfsPersistentVolumeSource(glusterfs *corev1.GlusterfsPersistentVolumeSource, w PrefixWriter) {
+	endpointsNamespace := ""
```
Comment (liggitt) [inline staging/src/k8s.io/kubectl/pkg/describe/versioned/describe.go]: prefer `<unset>`, see https://github.com/kubernetes/community/blob/master/contributors/devel/sig-cli/kubectl-conventions.md#output-conventions:

> For fields that can be explicitly unset (booleans, integers, structs), the output should say `<unset>`

Comment (liggitt) [reply staging/src/k8s.io/kubectl/pkg/describe/versioned/describe.go]: can you also add a unit test for the unset and set cases?

Comment (zouyee) [reply staging/src/k8s.io/kubectl/pkg/describe/versioned/describe.go]: done!

**Pull request merged**